easybuild/easyconfigs/h/HDF/HDF-4.2.15-GCCcore-10.2.0.eb

```diff
@@ -15,7 +15,21 @@ toolchainopts = {'pic': True}
 
 source_urls = ['http://www.hdfgroup.org/ftp/HDF/releases/HDF%(version)s/src/']
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['dbeeef525af7c2d01539906c28953f0fdab7dba603d1bc1ec4a5af60d002c459']
+patches = [
+    'HDF-4.2.15-GCCcore-10.2.0-ppc.patch', 
+    'HDF-4.2.15-GCCcore-10.2.0-sparc.patch', 
+    'HDF-4.2.15-GCCcore-10.2.0-s390.patch', 
+    'HDF-4.2.15-GCCcore-10.2.0-arm.patch', 
+    'HDF-4.2.15-GCCcore-10.2.0-aarch64.patch'
```

Comment:
@rungitta It would be nicer to combine these into a single big patch, but only for relevant stuff (ppc and aarch64, I think), and add a small header at the top of the patch to summarize what it does + where it comes from